avatar/avatar.py

```
@@ -95,20 +94,20 @@ async def create_session(self) -> Any:
+
+    async def start(self, agent_session: AgentSession, livekit_url: str, room: Any) -> bool:
         try:
             self._session = await self.create_session()
             if self._session is None:
                 return False
-                
-            await self._session.start(agent_session, room=room)
+
+            await self._session.start(agent_session, livekit_url=livekit_url, room=room)
             logger.info("Beyond Presence avatar started successfully")
             return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)